docs/user/openstack/etcd-ephemeral-disk.md

```diff
@@ -0,0 +1,252 @@
+# Moving etcd to an ephemeral local disk
+
+You can move etcd from a root volume (Cinder) to a dedicated ephemeral local disk to prevent or resolve performance issues.
+
+## Prerequisites
+
+* This migration is currently tested and documented as a day 2 operation.
+* An OpenStack cloud where Nova is configured to use local storage for ephemeral disks. The `libvirt.images_type` option in `nova.conf` must not be `rbd`.
+* An OpenStack cloud with Cinder being functional and enough available storage to accommodate 3 Root Volumes for the OpenShift control plane.
+* OpenShift will be deployed with IPI for now; UPI is not yet documented but technically possible.
+* The control-plane machine’s auxiliary storage device, such as /dev/vdb, must match the vdb. Change this reference in all places in the file.
+
+## Procedure
+
+* Create a Nova flavor for the Control Plane which allows 10 GiB of Ephemeral Disk:
+
+```bash
+openstack flavor create --ephemeral 10 [...]
+```
+
+* We will deploy a cluster with Root Volumes for the Control Plane. Here is an example of `install-config.yaml`:
+
+```yaml
+[...]
+controlPlane:
+  name: master
+  platform:
+    openstack:
+      type: ${CONTROL_PLANE_FLAVOR}
+      rootVolume:
+        size: 25
+        types:
+        - ${CINDER_TYPE}
+  replicas: 3
+[...]
+```
+
+* Run openshift-install with the following parameters to create the cluster:
+
+```bash
+openshift-install create cluster --dir=install_dir
+```
+
+* Once the cluster has been deployed and is healthy, edit the ControlPlaneMachineSet (CPMS) to add the additional block ephemeral device that will be used by etcd:
+
+```bash
+oc patch ControlPlaneMachineSet/cluster -n openshift-machine-api --type json -p '[{"op": "add", "path": "/spec/template/machines_v1beta1_machine_openshift_io/spec/providerSpec/value/additionalBlockDevices", "value": [{"name": "etcd", "sizeGiB": 10, "storage": {"type": "Local"}}]}]'
+```
+
+NOTE: Putting etcd on a block device of type Volume is not supported for performance reasons.
```

Comment:
If you want the note to stand out, you can use [the github additions to markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts):
```suggestion
> [!NOTE]
> Putting etcd on a block device of type Volume is not supported for performance reasons.
```

Now that raises the question of how etcd on an additional volume on ceph is different from keeping etcd on the root volume on ceph, and why it wouldn't be supported.

Reply:
I'll add a few words and use the new NOTE syntax.